docs/resources/migration.rst

```diff
@@ -230,22 +230,16 @@ Making things work with Pytest and caplog
 
 If you've followed all the migration guidelines thus far, you'll notice that this test will fail. This is because |pytest|_ links to the standard library's ``logging`` module.
 
-So to fix things, we need to add a sink that propagates Loguru to ``logging``.
-This is done on the fixture itself by mokeypatching |caplog|_. In your ``conftest.py`` file, add the following::
+So to fix things, we need to add a sink that propagates Loguru to the caplog handler.
+This is done by overriding the |caplog|_ fixture to capture its handler. In your ``conftest.py`` file, add the following::
 
-    import logging
     import pytest
-    from _pytest.logging import caplog as _caplog
```

Comment:
I didn't think this was right. I believe this works because Pytest injects `_caplog` before exporting a new `caplog` fixture (the one we define below). In fact, I think even before this import didn't actually do anything.

Reply:
You're probably right. The issue is that it generates a `fixture '_caplog' not found` error.

What do you think of overriding the existing `caplog` directly?

```python
@pytest.fixture
def caplog(caplog):
    handler_id = logger.add(caplog.handler, format="{message}")
    yield caplog
    logger.remove(handler_id)
```

Reply:
For some reason I thought the way to override system-level fixtures was to prefix with an underscore. I now see that isn't right - should just request the fixture itself and simply re-export it.

I also did you 1 better and type-hinted what `caplog` actually is 😉 

I tested in my project and it works great!